msteams-platform/sbs-api-msg-ext-ttk.yml

```diff
@@ -174,27 +174,27 @@ items:
 
     # [Command line](#tab/cline)
 
-    1. Sign in to Microsoft 365 with the TeamsFx CLI:
+    1. Sign in to Microsoft 365 with the Teamsapp CLI:
 
         ```cmd
 
-        teamsfx account login m365
+        teamsapp account login m365
 
         ```
 
         Your default web browser opens to let you sign in to the account. Sign in to your Azure account using your credentials. Close the browser when you're prompted.
 
-    2. Sign in to Azure with the TeamsFx CLI:
+    2. Sign in to Azure with the Teamsapp CLI:
 
         ```cmd
 
-        teamsfx account login azure
```

Comment:
Please check if we are using CLI in any steps by step guides.

Reply:
Sure, will check.